awx/main/tests/functional/api/test_inventory.py

```diff
@@ -609,21 +609,6 @@ def test_get_constructed_inventory(self, constructed_inventory, admin_user, get)
         r = get(url=reverse('api:constructed_inventory_detail', kwargs={'pk': constructed_inventory.pk}), user=admin_user, expect=200)
         assert r.data['update_cache_timeout'] == 53
 
-    def test_patch_constructed_inventory(self, constructed_inventory, admin_user, patch):
-        inv_src = constructed_inventory.inventory_sources.first()
-        assert inv_src.update_cache_timeout == 0
-        assert inv_src.limit == ''
-        r = patch(
-            url=reverse('api:constructed_inventory_detail', kwargs={'pk': constructed_inventory.pk}),
-            data=dict(update_cache_timeout=54, limit='foobar'),
-            user=admin_user,
-            expect=200,
-        )
-        assert r.data['update_cache_timeout'] == 54
-        inv_src = constructed_inventory.inventory_sources.first()
-        assert inv_src.update_cache_timeout == 54
-        assert inv_src.limit == 'foobar'
```

Comment:
Could you add some justification for deleting this? What exactly is being lost here?

Reply:
I hadn't meant to commit this- Thank you for catching it! This has been added back in here: 44dbc3272fa6fc54cc62ec058e26d091eeef527e